pkg/ebpf/probes/probes.go

```diff
@@ -7,26 +7,24 @@ import (
 	bpf "github.com/aquasecurity/libbpfgo"
 )
 
+// Probes
+
 type Probes interface {
 	Attach(probe Handle) error
 	Detach(probe Handle) error
+	DetachAll() error
+	DisableNetAutoload() error
 }
 
 type probes struct {
-	probes map[Handle]*probe
-	module *bpf.Module
-}
-
-type probe struct {
-	event     string
-	probeType probeType
-	fn        string
-	link      *bpf.BPFLink
+	module      *bpf.Module
+	probes      map[Handle]*probe
+	loadNetwork bool
```

Comment:
<img width="374" alt="image" src="https://user-images.githubusercontent.com/7395852/174138932-e8a394a7-0990-43b1-b0d4-f74f0e706d77.png">
<img width="731" alt="image" src="https://user-images.githubusercontent.com/7395852/174138980-27aa8a03-e98f-40ca-8b00-053290768aaf.png">

Reply:
Im marking this as solved because the existence or not of this variable will depend on what we conclude in the previous comment (about autoloading network probes by default, during initialization, or not).